xml/slemicro_security_selinux.xml

```diff
@@ -159,6 +159,93 @@
+    You may want a possibility to allow access to these directories or on
+    contrary restrict just some ports to the container even if SELinux is used
+    on your system. In this case, you need to create new policy rules that
+    enable or disable the access. &productnameshort; provide the Udica tool for
+    this purpose.
+   </para>
+   <para>
+    The following procedure describes how to create a custom policy for your
+    containers:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Make sure that SELinux is in the enforcing mode. For details, refer to
+      <xref linkend="sec-switching-modes"/>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Start a container using the following parameters:
+     </para>
+<screen>&prompt.root;podman run -v /home:/home:rw -v /var/:/var/:rw -p 21:21 -it sle15 bash</screen>
+     <para>
+      The container runs with the default policy that does not allow access to
+      the mount points, but does not restrict other ports.
+     </para>
+    </step>
+    <step>
+     <para>
+      You can exit the container.
+     </para>
+    </step>
+    <step>
+     <para>
+      Obtain the container ID:
+     </para>
+<screen>&prompt.root;podman ps -a
+
+CONTAINER ID  IMAGE                                                                              COMMAND     CREATED        STATUS            PORTS               NAMES
+e59f9d0f86f2  registry.opensuse.org/devel/bci/tumbleweed/containerfile/opensuse/bci/ruby:latest  /bin/bash   8 minutes ago  Up 8 seconds ago  0.0.0.0:21->21/tcp  zen_ramanujan
+        </screen>
+    </step>
+    <step>
+     <para>
+      Create a JSON file that Udica will use to create a custom policy for the
+      container:
+     </para>
+<screen>&prompt.root;podman inspect e59f9d0f86f2 > <replaceable>OUTPUT_JSON_FILE</replaceable></screen>
+     <para>
+      For example, substitute <replaceable>OUTPUT_JSON_FILE</replaceable> with
+      <filename>container.json</filename>
+     </para>
+    </step>
+    <step>
+     <para>
+      Run Udica to generate a policy according to the container parameters:
+     </para>
+<screen>&prompt.root;udica -j <replaceable>OUTPUT_JSON_FILE</replaceable>  <replaceable>CUSTOM_CONTAINER_POLICY</replaceable></screen>
+     <para>
+      For example:
+     </para>
+<screen>&prompt.root;udica -j container.json custom_policy</screen>
+    </step>
+    <step>
+     <para>
+      According to the provided instructions, load the policy modules by
+      running:
+     </para>
+<screen>&prompt.root;semodule -i custom_policy.cil /usr/share/udica/templates/{base_container.cil,net_container.cil,home_container.cil}</screen>
+    </step>
+    <step>
+     <para>
+      Run a container with the new policy module by using the
+      <option>--security-opt</option> option as follows:
+     </para>
+<screen>&prompt.root;podman run --security-opt label=type:custom_policy.process e59f9d0f86f2</screen>
```

Comment:
the command should be something like  (adjust to **sle15** )
`podman run --security-opt label=type:my_container.process -v /home:/home:ro -v /var/spool:/var/spool:rw -p 21:21 -it fedora bash `
otherwise podman will try pull image e59f9d0f86f2 that doesn't exist